main.go

```diff
@@ -191,9 +172,9 @@ func main() {
 				},
 			},
 			{
-				Name:    "sign",
+				Name:    "stamp",
```

Comment:
Definitely a nit, but sign seems more common in lexicon than stamp? Are people going to know what that means?

Reply:
I thought it was disingenuous to call something "sign" if the thing produced isn't a signature. I think folks will know what "stamp" means? I can definitely rename or add an alias in the future.